src/views/UserInfoWrapper/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { useOktaAuth } from '@okta/okta-react';
+
+import { setUserGroups } from 'reducers/authReducer';
+
+type UserInfoWrapper = {
+  children: React.ReactNode;
+};
+
+const UserInfoWrapper = ({ children }: UserInfoWrapper) => {
+  const dispatch = useDispatch();
+  const { authState, authService } = useOktaAuth();
+
+  const storeUserInfo = async () => {
```

Comment:
For now, I am just storing user groups. We could also store info from the id token in the future (name, email, etc).